wiki/Tournaments/UKCC/X/en.md

```diff
@@ -6,13 +6,610 @@ tags:
+  3. [Silentroom - Nhelv (Nyxa) [Shademphrik's Insane]](https://osu.ppy.sh/beatmapsets/917915#osu/2034488)
+- DoubleTime
+  1. [Bangsili - Ah! Salvia (DJ Cheori Remix Ver.) (ykcarrot) [Insane]](https://osu.ppy.sh/beatmapsets/20403#osu/71332)
+  2. [doriko feat. Hatsune Miku - Paradise Cage (KanbeKotori) [Insane]](https://osu.ppy.sh/beatmapsets/23483#osu/80384)
+  3. [fhana - Hoshikuzu no Interlude (VINXIS) [Stardust]](https://osu.ppy.sh/beatmapsets/338635#osu/749197)
+
+### Qualifiers
+
+**[Download the mappack here (60 MB)](https://osuhq.ml/ukcc/ukccx-qual.zip)**
+
+- NoMod
+  1. [Ayane - FaV -F\*\*\* and Vanguard- (Corinn) [Happiness]](https://osu.ppy.sh/beatmapsets/494460#osu/1052871)
+  2. [Festa - Lemuria (QuiescentRabbit) [Elcheer's Extra]](https://osu.ppy.sh/beatmapsets/835474#osu/1766008)
+  3. [mafumafu feat. IA - Adagaeshi Syndrome (DJ Lucky) [RLC's Extra]](https://osu.ppy.sh/beatmapsets/938437#osu/2054203)
+- Hidden
+  1. [Pandize - Stratos (Mun) [Pandize's Extra]](https://osu.ppy.sh/beatmapsets/717623#osu/1515960)
+  2. [Conagusuri - Hina Choco Dark Matter (Mafiamaster) [Lunatic]](https://osu.ppy.sh/beatmapsets/15576#osu/56337)
+- HardRock
+  1. [Momo Belia Deviluke (CV: Toyosaki Aki) - MORE & MORE (- Ex -) [Kowari's Extra]](https://osu.ppy.sh/beatmapsets/1264403#osu/2627977)
+  2. [kors k - smooooch (Card N'FoRcE) [Another]](https://osu.ppy.sh/beatmapsets/17912#osu/64674)
+- DoubleTime
+  1. [zts - the executioner (-kevincela-) [Insane]](https://osu.ppy.sh/beatmapsets/342360#osu/756752)
+  2. [Kano - Setsuna Plus (luxoDeh) [Insane]](https://osu.ppy.sh/beatmapsets/140379#osu/350646)
+
+### Pocket Picks
+
+See the [Pocket Picks sheet](https://docs.google.com/spreadsheets/d/1Jj9t9Fn_WoJmIYW6dGfDgL4djEZfAIMQcDgfV-mskmc/edit#gid=1724187283) for a breakdown of the participants Pocket Picks.
+
+## Match results
+
+### Final Round
+
+#### Day 17
+
+| Player 1 |  |  | Player 2 | Match link | Player status |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| **Kingling** ![][flag_GB] | **4** | 1 | ![][flag_GB] Malteser | [#1](https://osu.ppy.sh/community/matches/80493729) | *Malteser was eliminated* |
+| **Voss** ![][flag_GB] | **7** | 1 | ![][flag_GB] Kerantina | [#1](https://osu.ppy.sh/community/matches/80499466) |  |
+
+#### Grand Finals: Part 1
+
+The full results for this match can be found at this [link](https://osu.ppy.sh/community/matches/80587587).
+
+| Players | Score | Player status |
+| --: | :-: | :-- |
+| ![][flag_GB] Kingling | 4 |  |
+| ![][flag_GB] Voss | 0 |  |
+| ![][flag_GB] Kerantina | 0 | *Kerantina was eliminated* |
+
+#### Grand Finals: Part 2
+
+| Player 1 |  |  | Player 2 | Match link | Player status |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| **Voss** ![][flag_GB] | **9** | 6 | ![][flag_GB] Kingling | [#1](https://osu.ppy.sh/community/matches/80679009) | *Voss was eliminated* |
+
+### Round 8
+
+#### Day 15
+
+| Player 1 |  |  | Player 2 | Match link | Player status |
```

Comment:
"Outcome" instead of "Player status"

while the tables might look a little empty on the github preview, i think it should look ok on the website since the cells aren't boxed all the way around